octodns/provider/azuredns.py

```diff
@@ -0,0 +1,423 @@
+#
+#
+#
+
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.dns import DnsManagementClient
+from azure.mgmt.dns.models import ARecord, AaaaRecord, CnameRecord, MxRecord, \
+    SrvRecord, NsRecord, PtrRecord, TxtRecord, Zone
```

Comment:
Several of these imports are unused and failing the ci lint process. You can run what CI runs by doing `./script/cibuild`